feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt

```diff
@@ -138,7 +138,7 @@ class BottomSheetSessionsFragment : Fragment(R.layout.fragment_bottom_sheet_sess
             binding.isFiltered = uiModel.filters.isFiltered()
             binding.filteredSessionCount.isVisible = uiModel.filters.isFiltered()
             groupAdapter.update(sessions.map {
-                sessionItemFactory.create(it, sessionsViewModel)
```

Comment:
Can we use default arguments? (Will it error?)

Reply:
That's right, fixed it !
And also set default arguments ​​for other Factories that use the same parameters.